oaknational/Oak-Web-Application
# Feat/update colours

## Description

- Updates Oak Colours removing old colour scheme and replacing with new colours

## Issue(s)

Fixes #LESQ-437

## How to test

1. Go to https://deploy-preview-2092--oak-web-application.netlify.thenational.academy
2. Click on everything, see if any major breaks or colours that shouldn't be there 😅


## Checklist
